### PR TITLE
web: activity tab using activity agent

### PR DESCRIPTION
**Pull request by arthyn**
Fixes TLON-2189 by changing notifications to now use the same source and (mostly the same) rendering as the mobile.

PR Checklist
- [ ] Includes changes to desk files
- [ ] Describes how you tested the PR locally (test ship vs livenet)
- [ ] If a new feature, includes automated tests
- [ ] Comments added anywhere logic may be confusing without context

### CODE REVIEW

File: apps/tlon-web/src/notifications/ActivitySummary.tsx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    const message = `replied in a thread you're involved in`;
+    return (
+      <SummaryMessageWrapper>
+        {Authors}
+        {` ${message}`}
+      </SummaryMessageWrapper>
+    );
+  }
+
+  if (relevancy === 'postToChannel') {
```
Comment (patosullivan) [inline apps/tlon-web/src/notifications/ActivitySummary.tsx:170]: We'll need to cover flag-post and flag-reply cases as well

**Review by patosullivan — CHANGES_REQUESTED** [review]
Looks good, only missing piece, I think, is rendering flagged posts/replies.

File: desk/app/activity.hoon [hunk 2]
```diff
@@ -857,8 +864,9 @@
   =/  volume  (get-volume:evt volume-settings -.event)
   ::TODO  support other event types
   =*  is-msg  ?=(?(%dm-post %dm-reply %post %reply) -<.event)
-  =*  supported
-    |(is-msg ?=(?(%dm-invite %chan-init) -<.event))
+  =*  inits  ?=(?(%dm-invite %chan-init) -<.event)
+  =*  flags  ?=(?(%flag-post %flag-reply) -<.event)
```
Comment (Fang-) [inline desk/app/activity.hoon:868]: Consider `is-init` and `is-flag` for consistency and clarity. (Arguably, `is-supported` too, but that one was already there.)

**Review by Fang- — COMMENTED** [review]
Hoon side looks good! A nit, but not a blocker.

**Review by latter-bolden — APPROVED** [review]
New virtualization stuff lgtm!

**Comment by jamesacklin** [comment]
Overall looks good and matches the content on mobile.

Clicking "Mark everything as read" unfortunately results in a spinner that hangs out forever with no meaningful dojo or js-console output. Is there something else I should be inspecting to see progress there?

image

**Pull request merged**